app/src/main/java/com/framgia/fsalon/screen/customer/CustomerPresenter.java

```diff
@@ -1,15 +1,36 @@
 package com.framgia.fsalon.screen.customer;
 
+import com.framgia.fsalon.data.model.CustomerResponse;
+import com.framgia.fsalon.data.source.CustomerRepository;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
+import io.reactivex.observers.DisposableObserver;
+import io.reactivex.schedulers.Schedulers;
+
+import static com.framgia.fsalon.utils.Constant.ApiParram.FIRST_PAGE;
+import static com.framgia.fsalon.utils.Constant.OUT_OF_INDEX;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.utils.Constant.OUT_OF_INDEX. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports